helm/cluster-cloud-director/templates/_kubeadm.tpl

```diff
@@ -0,0 +1,17 @@
+# The helper functions here can be called in templates and _helpers.tpl
+# This file should be self-sufficient. Don't call any functions from _helpers.tpl
+
+{{- define "kubeadmFiles" -}}
+- path: /run/kubeadm/gs-update-kubeadm-yaml-skip-kube-proxy.sh
+  permissions: "0700"
+  content: |
+    {{- .Files.Get "files/run/kubeadm/gs-update-kubeadm-yaml-skip-kube-proxy.sh" | nindent 4 }}
+- path: /run/kubeadm/gs-kube-proxy-config.yaml
+  permissions: "0600"
+  content: |
+    {{- .Files.Get "files/run/kubeadm/gs-kube-proxy-config.yaml" | nindent 4 }}
+- path: /run/kubeadm/gs-kube-proxy-patch.sh
+  permissions: "0700"
+  content: |
+    {{- .Files.Get "files/run/kubeadm/gs-kube-proxy-patch.sh" | nindent 4 }}
```

Comment:
Why do we still need `gs-kube-proxy-config.yaml` ?

Reply:
🤦